packages/shylu/shylu_node/tacho/src/impl/Tacho_NumericTools_LevelSet.hpp

```diff
@@ -2505,7 +2517,7 @@ class NumericToolsLevelSet : public NumericToolsBase<ValueType, DeviceType> {
            &buffer_size_L, (void*)buffer_L.data());
       } else {
         status =
-          #if ROCM_VERSION >= 50400
+          #if (ROCM_VERSION >= 50400 && ROCM_VERSION < 60000)
```

Comment:
Looks like this could be made more consistent with similar changes by swapping the order of the check and function api, e.g. change to:
```
      #if (ROCM_VERSION >= 60000)
      rocsparse_spmv
      #else
      rocsparse_spmv_ex
```